### DISCUSSION TITLE
q-select filter bug, list blank

### DISCUSSION THREAD

**Discussion by dike321** [quasarframework/quasar#14254] [General - Components / Directives / etc] [answered]
### What happened?

this issue is happening again, I use a filter on the input

and this code i write on jsFiddle

my quasar version
![image](https://user-images.githubusercontent.com/50574844/186362208-55753371-d68e-472a-a5c4-16817da3ac35.png)

### What did you expect to happen?

i expect dropdown not blank for list

### Reproduction URL

https://jsfiddle.net/48bv5o7u/

### How to reproduce?

https://user-images.githubusercontent.com/50574844/186358944-9a1c84e4-7891-442f-90d9-804147eadc07.mp4

### Flavour

Quasar CLI (@quasar/cli | @quasar/app)

### Areas

Components (quasar)

### Platforms/Browsers

Chrome

**Comment by pdanpdan** [comment, marked as answer]
I would need a reproduction for this.
You upgraded from Qv1.15.0 to Qv1.19.5 and it started to happen?
Can you try some intermediate versions and see when it is happening and when not?

**Reply by dike321** [reply]
yes thank you, this issue when upgrade quasar is not happen again
